swag/swag_client.py

```diff
@@ -116,6 +131,38 @@ async def process(self, message):
                 "modération. Ceci était un message de la Fédération Bobbyique du "
                 "Minage*"
             )
+        except NoCagnotteRegistered as e:
+            await message.channel.send(
+                f"Aucune €agnotte n°€{e.name} est active dans la $wagBank ! "
+                f"{message.author.mention}, tu t'es sans doute trompé de numéro 🤨"
+            )
+        except NoCagnotteIdxInCommand as e:
+            await message.channel.send(
+                "Aucun numéro de cagnotte n'est mentionné avec la forme €n (ex : €1)"
+                f"dans ta commande {message.author.mention}. 🤔"
+            )
+        except CagnotteNameAlreadyExist:
+            await message.channel.send(
+                f"{message.author.mention}, une €agnotte porte déjà ce nom ! "
+                "Je te conseille de choisir un autre nom avant que tout le monde "
+                "soit complètement duper 🤦‍♂️"
+            )
+        except NotEnoughMoneyInCagnotte:
+            await message.channel.send(
+                f"{message.author.mention}, tu es entrain de demander à une €agnotte "
```

Comment:
en train

Reply:
Resolved in https://github.com/BobbySoftwares/Bybbot/pull/7/commits/173d6bc07cf9a1a3121d5820f16d9bcb19a74a16